### ISSUE TITLE
`polkadot --version` check for `execute` and `prepare` binary

### ISSUE THREAD

**Issue by ggwpez** [paritytech/polkadot-sdk#6671]
This is more of Q and suggestion: maybe the version output of the polkadot binary should also do a light self-check to ensure that the execute and prepare worker binaries exist and have the expected version.

Someone in the validator lounge posted this which got me thinking:
>I m now concerned about the installation (polkadot-execute-worker and polkadot-prepare-worker returns command not found whereas polkadot --version returns polkadot 1.16.2-dba2dd59101) ...

**Comment by bkchr** [comment]
These version checks are done at startup of the node. It checks if the workers exist and if they have the correct version. So, I don't see a reason that we do this check as part of `version`. Maybe it could be its own check `check-installation`, but I don't see that much value in this if the node refuses to work if it can not find the workers any way. The node also provides the option to pass the paths of the workers manually, which would bring up the question, should the extra check do this as well, but then people may forget to pass these paths to the node launch etc. 

TLDR: I think this is not actionable/required right now.
